<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ViewButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>查看原图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TransferButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>360</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>变换</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PathEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>471</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>C:/Users/richardchen0714/Documents/Gray/</string>
    </property>
   </widget>
   <widget class="QLabel" name="Path">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>文件路径</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CenterEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Center">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>窗位</string>
    </property>
   </widget>
   <widget class="QLabel" name="Width">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>窗宽</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="WidthEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>250</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
